Header header

uint8 horizontal_flag
float32 horizontal_rate
uint8 vertical_flag
float32 vertical_rate
uint8 kurukuru